plugin/net/cni.go

```diff
@@ -70,28 +77,31 @@ func (c *CNIPlugin) CmdAdd(args *skel.CmdArgs) error {
 	if err != nil {
 		return fmt.Errorf("unable to allocate IP address: %s", err)
 	}
+	// Only expecting one address
+	ip := result.IPs[0]
```

Comment:
This is a valid question.  However I am not sure we can do much better:
* `getIP` errors on zero addresses.
* there is no mechanism to log a warning in CNI.